todo_app/data/item.py

```diff
@@ -0,0 +1,73 @@
+from todo_app.config.credentials import *
```

Comment:
From the [PEP 8 style guide](https://peps.python.org/pep-0008/):

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools

Basically, it's not clear when reading the code that `api_token` came from the credentials module, and you might accidentally define new variables here that reuse the names etc.

Reply:
Fixed